NetCoreRestApi/BusinessLayer/Manager/ProductManager.cs

```diff
@@ -1,17 +1,19 @@
-﻿using BusinessLayer.Models;
+﻿using DataLayer.Models;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace BusinessLayer.Manager
 {
     public class ProductManager : IProductManager
     {
-        public IEnumerable<ProductModel> GetAll()
+        public IQueryable<ProductModel> GetAll()
         {
+            var ID = 1;
             var productModels = new List<ProductModel>();
-            productModels.Add(new StubProductModel("first product"));
-            productModels.Add(new StubProductModel("second product"));
-            productModels.Add(new StubProductModel("third product"));           
-            return productModels;
+            productModels.Add(new StubProductModel(ID, "first product"));
+            productModels.Add(new StubProductModel(++ID, "second product"));
+            productModels.Add(new StubProductModel(++ID, "third product"));           
```

Comment:
Replace StubProductModel with ProductModel class

Reply:
Ok, I will try to change it

Reply:
done